### PR TITLE
Use config value for endpoint in Worker.js script

### LINKED ISSUE #64: Make boostrap peer list configurable via the config/*.json files
### Problem

At the moment the peers to connect to in the auth lobby shared worker are hard-coded for production and staging nodes here:

https://github.com/fission-suite/auth-lobby/blob/9234e66752946f74c618e9dd4a2b0fd095cf8199/src/Javascript/Worker.js#L105-L107

It would be great if they were configurable via the `config/*.json` files, too, so I can e.g. easily support connecting up my pizza instance/other people can configure their own fission constellation without code changes.

### Solution

I'm not quite sure. I know that it's not as easy as with the other variables, as there's no html file for the worker for mustache to inject the variables into. Maybe we can do something with esbuild's --define parameters + some [`jq`](https://stedolan.github.io/jq/) querying?

### PR DESCRIPTION
**Pull request by matheus23**
Fixes #64

### CODE REVIEW

File: Justfile [hunk 1]
```diff
@@ -90,7 +90,11 @@ insert-version:
 	rm -rf {{dist_dir}}/web_modules
 	cp -rf web_modules {{dist_dir}}/web_modules
 	cp {{src_dir}}/Javascript/Main.js {{dist_dir}}/index.js
-	{{node_bin}}/esbuild --bundle --outfile={{dist_dir}}/worker.min.js {{src_dir}}/Javascript/Worker.js
+	{{node_bin}}/esbuild \
+		--bundle \
+		--define:API_ENDPOINT="$(jq .API_ENDPOINT config/{{config}}.json)" \
```
Comment (matheus23) [inline Justfile:95]: Here's where the magic happens:
`jq` will pull a json value specified by the path out of a json file. `.` refers to the json's root, and then we look for the `API_ENDPOINT` key.

Then esbuild will replace `API_ENDPOINT` in the js source with the value returned from `jq` (note that jq will return the value quoted). [Also, here's a link to esbuild's define docs](https://esbuild.github.io/api/#define)

**Review by matheus23 — COMMENTED** [review]
I'm not happy with this. I'm opening this just so it can be discussed/improved upon.

At the moment, running `just production-build` will not work correctly :exclamation:
You need to use `just config=production production-build` and that's kind of repetitive (and error-prone). I'm open for suggestions. Can we maybe even get rid of `production-build` and `staging-build` etc. and make it possible to just use `just config=production build`?

**Review by icidasset — APPROVED** [review]
> You need to use `just config=production production-build` and that's kind of repetitive (and error-prone).

If you the `js` task from the `production-build` task dependencies (ie. the tasks after the task name). And move it to the `just` call in the task itself, then it should work.

But yeah I agree, this could be better.

In any case 👍

**Comment by matheus23** [comment]
Ah nice. I don't want to mess up the order of these calls, so I'll probably move all of the calls into the body of the task, that should do the same thing, right?

**Comment by icidasset** [comment]
> Ah nice. I don't want to mess up the order of these calls, so I'll probably move all of the calls into the body of the task, that should do the same thing, right?

That works too 👍

**Pull request merged**